### DISCUSSION TITLE
Copilot Payment

### DISCUSSION THREAD

**Discussion by vincenzo-pesticcio** [community/community#61280] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi, I asked for copilot to be deactivated this morning but a paypal payment still took place tonight, can I get a refund?

**Comment by davecheney** [comment, marked as answer]
Please contact support, https://support.github.com/contact.

**Comment by beuuett-lim** [comment]
I want to pay copilot x by changing my card.
There is a previous cancellation history, so it is only canceled and I cannot payment cannot be made with a new card.

**Reply by davecheney** [reply]
Please contact support, https://support.github.com/contact.
